editoast/src/core/simulation.rs

```diff
@@ -38,22 +39,21 @@ editoast_common::schemas! {
 pub struct PhysicsConsist {
     pub effort_curves: EffortCurves,
     pub base_power_class: Option<String>,
-    /// Length of the rolling stock in mm
-    pub length: u64,
-    /// Maximum speed of the rolling stock in m/s
-    #[derivative(Hash(hash_with = "editoast_common::hash_float::<3,_>"))]
-    pub max_speed: f64,
+    /// Length of the rolling stock
+    #[derivative(Hash(hash_with = "editoast_common::hash_length::<3,_>"))]
+    pub length: Length,
+    /// Maximum speed of the rolling stock
+    #[derivative(Hash(hash_with = "editoast_common::hash_velocity::<3,_>"))]
+    pub max_speed: Velocity,
     // Time in ms
     pub startup_time: u64,
-    #[derivative(Hash(hash_with = "editoast_common::hash_float::<5,_>"))]
-    /// In m/s²
-    pub startup_acceleration: f64,
-    #[derivative(Hash(hash_with = "editoast_common::hash_float::<5,_>"))]
-    /// In m/s²
-    pub comfort_acceleration: f64,
-    #[derivative(Hash(hash_with = "editoast_common::hash_float::<5,_>"))]
+    #[derivative(Hash(hash_with = "editoast_common::hash_acceleration::<5,_>"))]
+    pub startup_acceleration: Acceleration,
+    #[derivative(Hash(hash_with = "editoast_common::hash_acceleration::<5,_>"))]
+    pub comfort_acceleration: Acceleration,
     /// The constant gamma braking coefficient used when NOT circulating
     /// under ETCS/ERTMS signaling system in m/s^2
+    #[derivative(Hash(hash_with = "editoast_common::hash_float::<5,_>"))]
     pub const_gamma: f64,
     #[derivative(Hash(hash_with = "editoast_common::hash_float::<5,_>"))]
     pub inertia_coefficient: f64,
```

Comment:
Shouldn't `intertia_coefficient` and `const_gamma` be of some units too? Not sure if they are ratio or actual units 🤷